### DISCUSSION TITLE
First Steps for Contributing

### DISCUSSION THREAD

**Discussion by eninkey** [Arcticons-Team/Arcticons#337] [Q&A]
Yesterday I created an icon and tried to contribute it.
I followed the instructions, but I'm not sure if it's right. 
* Fork the repository to my GitHub account
* and create a pull request (in my fork)

Do you notice any changes/pull request in my fork?

Also tried to update the appfilter.xml file.

**Comment by Aman9das** [comment]
I agree we should ramp up the contributing guide.. Thanks for the contribution!

Read the ending part of this:
https://github.com/Aman9das/Arcticons/blob/candybar/Vector%20Asset%20Creator.md

**Reply by eninkey** [reply]
Thank you. Yes, when using GitHub the first time, it's not that clear.
Now it should be right. I hope :D

Do the svg have to have the "drawable" form the appfilter.xml as name?

And it’s written that I should use template files as a reference: There are two layers in the template files, but in the finale svg-file the background layer should be removed, right?

**Reply by Aman9das** [reply]
I dont know i dont look at templates :p

>Do the svg have to have the "drawable" form the appfilter.xml as name?

Yes edit appfilter.xml to drawable="svg_file_name"

**Reply by eninkey** [reply]
> I dont know i dont look at templates :p

Ok, but only the white lines in the svg is how it should be, right?

**Reply by Aman9das** [reply]
Yes.. the stroke width is supposed to be 1 pixel.. dont worry though, @Donnnno will look at your icons and make improvements/fixes.

So take it casually at your end.. :)

**Reply by Donnnno** [reply]
I recommend using or at least starting with a template file. The artboards are already the right size and use the correct line-thickness and cap & corner settings.

**Comment by hassanncreative** [comment]
I have a question. How can we update an icon which is already been released? I feel that one or two icons which I designed in the past needs some polishing.

**Reply by Donnnno** [reply]
Just create a new one with the same name and put it in the `/other` folder, it will be replaced with the next update.

**Reply by hassanncreative** [reply]
Okay. Thank you.

**Comment by hassanncreative** [comment]
![Screenshot 2022-02-10 191259](https://user-images.githubusercontent.com/85166900/153425123-f488bbfa-e08c-40a2-a282-34a17aa0bce6.jpg)

I just uploaded three icons in other folder and when I went ahead to create a pull request, the button is inactive and giving me this page. Never encountered this before. Can someone tell me what am I doing wrong?
